compiler/src/iree/compiler/InputConversion/StableHLO/Preprocessing/StableHLOToStableHLO.cpp

```diff
@@ -1799,6 +1834,7 @@ struct StableHLOToStableHLOPreprocessing final
 
     // Unary elementwise op.
     patterns.insert<
+        ZeroConcat,
```

Comment:
Maybe move this to l.1833? It is more like `additional canonicalization` patterns to me.

Reply:
Moved.